app/ide-desktop/lib/content/src/tailwind.css

```diff
@@ -1,3 +1,268 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* Copied from https://unpkg.com/tailwindcss@3.2.7/src/css/preflight.css. */
+/* This is required because these styles should not affect the IDE. */
+@layer base {
+  #dashboard *,
+  #dashboard ::before,
+  #dashboard ::after {
+    box-sizing: border-box; /* 1 */
+    border-width: 0; /* 2 */
+    border-style: solid; /* 2 */
```

Comment:
Why we are copy-pasting code from somewhere. This is a no-go in all of our repos so far, so please, let's find a way not to do it. Question to the person doing code review - why was it not caught?

Reply:
i've mentioned elsewhere - but it's difficult to avoid copy-pasting because tailwind does not support prefixes on its preflight. it may be possible using alternative approaches like tailwindcss/nesting and running postcss instead of tailwind

Reply:
By prefixes, you mean `#dashboard`? Anyway, we need to find a way to do it, copy-pasting generated things is really a no-go. If you want, we can have a call about it and chat about possible solutions.

Reply:
should be fixed in #6034. can probably be fixed here, *however* Create React App is incompatible with custom postcss processing ([see this link](https://tailwindcss.com/docs/guides/create-react-app)), so this will mean that it won't be possible to make the styles work for `dashboard/` (not that they are currently working)

Reply:
as a stopgap solution it might be better to just remove tailwind support completely from current develop, since a new implementation should be coming soon anyway

Reply:
Ok, in order not to block it, I'm OK with any solution. If you want to merge this file for now and we could fix it later, perfect, but lets add a comment about it there, please :)